lib/net.js

```diff
@@ -979,7 +976,7 @@ function lookupAndConnect(self, options) {
   var localAddress = options.localAddress;
   var localPort = options.localPort;
 
-  if (localAddress && !exports.isIP(localAddress))
+  if (localAddress && !cares.isIP(localAddress))
```

Comment:
Making `isIP` available as a variable would help simplify here and in `module.exports`.